articles/quickstart/backend/aspnet-core-webapi/01-authorization.md

```diff
@@ -54,45 +54,32 @@ In your application, register the authentication services:
 2. Make a call to the `AddJwtBearer` method to register the JWT Bearer authentication scheme. Configure your Auth0 domain as the authority, and your Auth0 API identifier as the audience. In some cases the access token will not have a `sub` claim which will lead to `User.Identity.Name` being `null`. If you want to map a different claim to `User.Identity.Name` then add it to `options.TokenValidationParameters` within the `AddAuthentication()` call.
 
 ```csharp
-// Startup.cs
+// Program.cs
 
-public void ConfigureServices(IServiceCollection services)
+var domain = $"https://{builder.Configuration["Auth0:Domain"]}/";
```

Comment:
Can we add what builder is here ? So let's add `var builder = WebApplication.CreateBuilder(args);`

Reply:
Done as per [501677e](https://github.com/auth0/docs/pull/10231/commits/501677e349cf36e85c17f6bd1c4a5c96d19ea441).